test/mocha/utils.coffee

```diff
@@ -44,8 +48,52 @@ describe 'NVD3', ->
       returnedFunction({},2).should.be.equal '#aaa'
       returnedFunction({},3).should.be.equal '#000'
 
+  describe 'Sanitize Height and Width for a Container', ->
+    it 'provides default height', ->
+      h = (arg) -> return null
+      cont = { style: h }
```

Comment:
For these tests you can create a real DOM element: document.createElement('div').  That way it better mimics how the method will be used.